notebook/static/base/js/i18nload.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Plugin to load a single locale.
+ */
+define([
+	"require",
+	"module",
+	 // These are only here so that the optimizer knows which ones we MIGHT load.
+	 // We will actually only load the ones we need. There should be one entry
+	 // here for each language you want to support.
+	 // For example, for German....
+	 // "json!base/../../i18n/de/LC_MESSAGES/nbjs.json"
+    ], function (require, module) {
+	return {
+		id: module.id,
+
+		load: function (locale, callerRequire, onload, loaderConfig) {

```

Comment:
Am I right that this function is not used? `i18n.js` appears to use `i18nload.id` and call require directly.